src/components/reports/ReportForm.tsx

```diff
@@ -85,7 +86,7 @@ const ReportForm = () => {
       <h1>Add a new report</h1>
       <Formik
         enableReinitialize={true}
-        initialValues={initialValues}
+        initialValues={props.initialValues}
```

Comment:
Entirely down to personal preference, you could deconstruct the props if desired.

Reply:
My default is to deconstruct props like so:

`const { initialValues } = props;`

When you only have one prop you could also deconstruct directly in the components argument. Either is fine.